Zotero/Models/AnnotationsConfig.swift

```diff
@@ -29,6 +29,13 @@ struct AnnotationsConfig {
     static let keyKey = "Zotero:Key"
     // Line width of image annotation in PDF document.
     static let imageAnnotationLineWidth: CGFloat = 2
+    // Free text annotation font size minimum, maximum, increment and rounding
+    static let freeTextAnnotationFontSizeMinimum: CGFloat = 1
+    static let freeTextAnnotationFontSizeMaximum: CGFloat = 200
+    static let freeTextAnnotationFontSizeIncrement: CGFloat = 0.5
+    static func roundFreeTextAnnotationFontSize(_ fontSize: CGFloat) -> CGFloat {
```

Comment:
Just out of curiosity, why is this needed? Did we have some weird fractions stored or does server send other values?

Reply:
If the user modifies the size using the resize handle, or even if they just start editing the text and do nothing, the size value changes. For conformity with the desktop client, that uses `0.5` steps, we round appropriately.